pages/bib/[id]/index.tsx

```diff
@@ -122,37 +131,58 @@ export default function BibPage({
       }
     )
     const bibQueryString = getBibQueryString(newQuery, false, viewAllItems)
+    try {
+      // Cancel any active fetches on new ItemTable refreshes
+      if (controllerRef.current) {
+        controllerRef.current.abort()
+      }
+      controllerRef.current = new AbortController()
+      const signal = controllerRef.current.signal
+      const response = await fetch(
+        `${BASE_URL}/api/bib/${bib.id}${bibQueryString}`,
+        {
+          method: "get",
+          signal,
+        }
+      )
+      if (response?.ok) {
+        const { discoveryBibResult } = await response.json()
+        setBib(new Bib(discoveryBibResult))
 
-    // Cancel any active fetches on new ItemTable refreshes
-    if (controllerRef.current) {
-      controllerRef.current.abort()
-    }
-    controllerRef.current = new AbortController()
-    const signal = controllerRef.current.signal
-
-    const response = await fetch(
-      `${BASE_URL}/api/bib/${bib.id}/items${bibQueryString}`,
-      {
-        method: "get",
-        signal,
+        setItemsLoading(false)
+        itemTableScrollRef.current?.scrollIntoView({
```

Comment:
is this scrolling behavior a current requirement or is it leftover from DFE? I believe we had the scrolling from when we were batch loading all the items from view all items.I think the experience is its potentially an accessibility issue and jarring to have the screen move and obscure the applied filters

Reply:
Good call out about obscuring the applied filters, but this was an accessibility suggested by Edwin and confirmed by Clare.

Reply:
what is the issue that the scroll into view solves?